webapp/src/Service/CheckConfigService.php

```diff
@@ -337,6 +346,36 @@ public function checkTmpdirWritable()
                  $tmpdir)];
     }
 
+    public function checkHashTime()
+    {
+        $tmpuser = new User();
+        $counter = 0;
+        $time_start = microtime(true);
+        do {
+            $plainPassword = 'ryanpass';
+            $encoded = $this->passwordEncoder->encodePassword($tmpuser, $plainPassword);
+            // $user->setPassword($encoded);
+            $time_end = microtime(true);
+            $counter++;
+        } while ( ($time_end - $time_start) < 4);
```

Comment:
This 4 is the number of seconds right? Maybe we should lower it a bit if that's the case? I guess that we can also determine a sane value in 2 seconds.

Reply:
I now delay the page for 4s, will change it to 2s

Reply:
Sounds good.